cpp/src/io/orc/reader_impl.hpp

```diff
@@ -34,7 +34,7 @@
 #include <utility>
 #include <vector>
 
-namespace cudf::io::detail::orc {
+namespace cudf::io::orc::detail {
 using namespace cudf::io::orc;
```

Comment:
We should not need this anymore, since now `cudf::io::orc::detail` is nested in `cudf::io::orc`.